server/api.js

```diff
@@ -43,4 +43,24 @@ router.get("/laptop_request", async (req, res) => {
 	}
 });
 
+router.post("/laptop_donation", (req, res) => {
+	// create a database table for 
+	let name = req.body.name;
+	let address = req.body.address;
+	let laptops = req.body.numberOfLaptops;
+	let phonenumber = req.body.phoneNumber;
+	let email = req.body.email;
+	let deliveryoption = req.body.deliveryOption;
+
+	const query =
+		" insert into laptop_donations (name, address, laptops, phonenumber, email, deliveryoption) values ($1, $2, $3, $4, $5, $6)";
```

Comment:
note that if you change names as suggested in sql, this is the only line that needs modifying in api.js